src/watchmaker/managers/worker.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
How about putting this in `workers/base.py`?

Reply:
I actually renamed base.py to be platform.py and separate out the three services (platform management, worker management, and worker) into their own files.

Reply:
Understood, but this is not a manager. It is a new base class, for workers. To me, it seems more natural to put it in the `workers.base` module.

Reply:
Ah, misunderstood what you were saying. That's my bad. In my mind I was thinking managers/base.py

Reply:
Done